src/mfa/interfaces/challenge.interface.ts

```diff
@@ -4,7 +4,7 @@ export interface Challenge {
   createdAt: string;
   updatedAt: string;
   expiresAt?: string;
-  code: string;
+  code?: string;
```

Comment:
note that this interface is also shared w/ standalone MFA. does standalone have a required code?

Reply:
oh it was made optional

Reply:
🤦‍♀️

Reply:
ignore me!!!